ndscan/fitting/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .fitting import FitBase  # noqa F401
```

Comment:
@lochsh any idea why this was needed? Otherwise flake8 was complaining, but it doesn't seem to complain about the same pattern in other places 🤔

Reply:
Isn't https://github.com/OxfordIonTrapGroup/ndscan/blob/d96251c72923b4493236cd70102cc3aa52ac375a/ndscan/experiment/__init__.py#L17-L29 the same thing? Looking at that again, I guess flake8 doesn't complain about that because we use the imports in the subsequent `__all__.extend` lines.

Reply:
I missed that one, sorry. But yes, because they're used in the file you don't get a complaint about them being imported but unused.

Reply:
https://github.com/OxfordIonTrapGroup/ndscan/pull/305/commits/7b6f536f2c9f73372ce32e47d19232c08f7e9ad5 makes this follow the practice in the rest of `ndscan`